build.gradle

```diff
@@ -123,17 +123,18 @@ subprojects {
 
                 // Test dependencies.
                 junit: 'junit:junit:4.11',
-                mockito: 'org.mockito:mockito-core:1.10.19'
+                mockito: 'org.mockito:mockito-core:1.10.19',
+                tcnative: 'io.netty:netty-tcnative:1.1.33.Fork3'
```

Comment:
That's because you have explicitly selected it with sslProvider(SslProvider.OPENSSL). Don't specify OpenSSL if you are trying to use NPN with alpnboot.